assets/src/components/d3/createHistogram.js

```diff
@@ -55,7 +55,7 @@ function createHistogram ({ data, width, height, domElement, xAxisLabel, yAxisLa
   main.append('div')
```

Comment:
@ssciolla I see a deprecation notification in this file for `d3.histogram` and instead it is suggesting to use `bin()`? How you consider this change or you want to do this as part of #1367

Reply:
I must have missed that. Is it a warning or an error? Likely `d3` is warning us that something will be deprecated in `v6`. If it's an easy swap, we could try it.

Reply:
I think VScode is looking at future version of d3 and webpack is still referring to v5.16. it is really strange that when my local node module has v5.16 downloaded, why VSCode try to present with a warning of deprecation for future version. I think this is not an issue with that version of d3 we reference.